### DISCUSSION TITLE
Change Unit of all CSS properties at once in StyleManager

### DISCUSSION THREAD

**Discussion by vamshi0** [GrapesJS/grapesjs#4478] [Q&A]
Hello everyone,
i want to change unit of all empty integer CSS properties at once,

i have tried,
const units = ["px", "%", "rem"];
const properties = editor.StyleManager.getProperties("dimension");
properties.map(property => property.set("unit", units[1]));

but it is selecting the property values and including them in the code,
margin: 0 0 0 0;
padding: 0 0 0 0;
border: 0;
border-radius:0;

is there any way to change units without selecting the property values ?

Thanks in Advance..!

**Comment by Amandasilvbr** [comment]
Hi, @vamshi0. 

If you use vscode, I strongly recommend doing it like this:

Press ctrl+shift+h

On the first line, write which word you want to replace, and on the second which it will be replaced. So properties like margin: 0 0 0 will not be selected, only the ones with "px", for example.

![image](https://user-images.githubusercontent.com/100382936/180779596-dbb62c46-a4ce-4ce4-aaa4-3bd8159e7bc8.png)

You you have any questions, let´s talk about!

**Reply by vamshi0** [reply]
not for VS code,

i added an option to change unit of every property in Grapesjs StyleManager.
So, that i can change all at once as per my selection.

----- example  code ----
const sm = editor.StyleManager;
const brProperty = sm.getProperty('decorations', 'border-radius')[1];
brProperty.get('properties').forEach(property => property.set('unit', 'rem'));
